builtin/logical/nomad/backend_test.go

```diff
@@ -14,6 +16,18 @@ import (
 	"github.com/ory/dockertest"
 )
 
+// randomWithPrefix is used to generate a unique name with a prefix, for
+// randomizing names in acceptance tests
+func randomWithPrefix(name string) string {
+	reseed()
```

Comment:
What is the `reseed` function for? It is seeding the global rand source, but then you're using a new random source in your `Sprintf`. You can alternatively also use an existing utility like base62.Random() to generate random strings.